assignment-section-05/tests/test_prediction.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+from classification_model.predict import make_prediction
+
+
+def test_make_prediction(sample_input_data, first_ten_expected_predictions):
+    # Given
+    expected_no_predictions = 131
+
+    # When
+    result = make_prediction(input_data=sample_input_data)
+
+    # Then
+    predictions = result.get("predictions")
+    assert isinstance(predictions, np.ndarray)
+    assert isinstance(predictions[0], np.int64)
+    assert result.get("errors") is None
+    assert len(predictions) == expected_no_predictions
+    assert list(predictions)[:10] == first_ten_expected_predictions
```

Comment:
we should probably corroborate the actual value of the predictions. We don't do this in the house price either, do we?

Reply:
Just had a look - predictions appear to be really off. Any idea why the model is so inaccurate?

Reply:
0.71 accuracy

Reply:
Most likely because we are using logistic regression. We should use trees for classification.